operational/azure/marketplace_new_products/azure_marketplace_new_products.pt

```diff
@@ -226,11 +226,11 @@ end
 datasource "ds_current_products_oldconnection" do
   iterate $ds_bill_connections_old
   request do
-    run_script $js_get_products_oldconnection, val(iter_item, 'id'), $ds_top_level_bcs, rs_org_id, rs_optima_host, 3
+    run_script $js_get_products_oldconnection, val(iter_item, 'id'), $ds_top_level_bcs, 3, rs_org_id, rs_optima_host
```

Comment:
I think this was fine before 

<blockquote> run_script statements found whose parameters are not in the correct order. run_script parameters should be in the following order: script, val(iter_item, string), datasources, parameters, variables, raw values:

Line 229: ds_current_products_oldconnection / run_script $js_get_products_oldconnection
<html>
<body>
<!--StartFragment--><markdown-accessiblity-table data-catalyst="" style="box-sizing: border-box; display: block; margin-top: 0px !important; color: rgb(240, 246, 252); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Noto Sans&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">
Line 325: ds_current_products_newconnection / run_script $js_get_products_newconnection
--


</markdown-accessiblity-table><br class="Apple-interchange-newline"><!--EndFragment-->
</body>
</html></blockquote>

```suggestion
    run_script $js_get_products_oldconnection, val(iter_item, 'id'), $ds_top_level_bcs, rs_org_id, rs_optima_host, 3
```
(Would obviously mean lines 243, 255, 325, 339 and 351 would need to be reverted)